### PR TITLE
CY-1889: adding pre-push hook for checking lint issues

### PR DESCRIPTION
**Pull request by tyacbovi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
   },
   "dependencies": {
     "better-console": "^0.2.4",
-    "blueimp-md5": "^2.7.0",
+    "blueimp-md5": "^2.7G.0",
```
Comment (qooban) [inline package.json]: Typo?

File: package.json [hunk 2]
```diff
@@ -149,5 +150,15 @@
     "webpack-cli": "^3.3.9",
     "webpack-dev-middleware": "^3.7.2",
     "webpack-dev-server": "^3.8.2"
+  },
+  "husky": {
```
Comment (qooban) [inline package.json]: Husky shouldn't be added to `devDependencies`?

Comment (tyacbovi) [reply package.json]: That's the git hook settings, not the package

**Comment by qooban** [comment]
Cleaning open PRs, not merging solution for now.

**Pull request closed without merging**